### PR TITLE
Fixes `at-rule-no-unknown` false positives for `@scroll-timeline`

### LINKED ISSUE #6548: Fix `at-rule-no-unknown` false positives for `@scroll-timeline`
### What steps are needed to reproduce the bug?

use `@scroll-timeline`
see https://caniuse.com/css-scroll-timeline

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "at-rule-no-unknown": "true"
  }
}
```

### Proposal to fix the bug

edit https://github.com/stylelint/stylelint/blob/main/lib/reference/atKeywords.js

### PR DESCRIPTION
**Pull request by mattxwang**
> Which issue, if any, is this issue related to?

Closes #6548.

> Is there anything in the PR that needs further explanation?

Pretty self-explanatory - added failing test case, then used @Mouvedia's proposed solution.

### CODE REVIEW

File: lib/rules/at-rule-no-unknown/__tests__/index.js [hunk 1]
```diff
@@ -91,6 +91,9 @@ testRule({
 		{
 			code: '@layer framework { h1 { background: white; } }',
 		},
+		{
+			code: '@scroll-timeline spin-slash { }',
```
Comment (Mouvedia) [inline lib/rules/at-rule-no-unknown/__tests__/index.js:95]: https://github.com/stylelint/stylelint/blob/797381ced3190aa0c78bff8fb892572caadf4305/docs/developer-guide/rules.md?plain=1#L167

Comment (mattxwang) [reply lib/rules/at-rule-no-unknown/__tests__/index.js]: Thanks - just updated the test case!

File: .changeset/nice-ligers-type.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixes `at-rule-no-unknown` false positives for `@scroll-timeline`
```
Comment (ybiquitous) [inline .changeset/nice-ligers-type.md:5]: [suggest] Please update the prefix:

```suggestion
Fixed: `at-rule-no-unknown` false positives for `@scroll-timeline`
```

See also:

https://github.com/stylelint/stylelint/blob/797381ced3190aa0c78bff8fb892572caadf4305/docs/maintainer-guide/pull-requests.md?plain=1#L22-L24

**Review by ybiquitous — APPROVED** [review]
Thank you, LGTM 👍🏼

File: .changeset/nice-ligers-type.md (hunk 2 above)
Comment (mattxwang) [reply .changeset/nice-ligers-type.md:5]: Ah - my mistake! I'll also adjust my other PR with this. Still new to the changeset tool!

**Pull request merged**